tools/toolchain_profiler.py

```diff
@@ -152,7 +152,7 @@ def escape_string(arg):
 
     @staticmethod
     def escape_args(args):
-      return map(lambda arg: ToolchainProfiler.escape_string(arg), args)
+      return (ToolchainProfiler.escape_string(arg) for arg in args)
```

Comment:
Just return a list here maybe?  `[ToolchainProfiler.escape_string(arg) for arg in args]`

Reply:
[`map()`](https://docs.python.org/3/library/functions.html#map) returns an iterator in Python 3 so the linter's fix wants to return a generator expression.  Lists take more RAM than maps and generators.

Reply:
Sure, this is far from memory critical code though and I think the explicit list is more idiomatic (at least to my old fasioned eyes).

Reply:
I will change this but...

The Py 2 to 3 transition contained a few key transitions.
* `bytes == str` vs. `bytes != str`
* https://docs.python.org/3/whatsnew/3.0.html#print-is-a-function (and related changes)
* https://docs.python.org/3/whatsnew/3.0.html#views-and-iterators-instead-of-lists

Lists were more idiomatic in in legacy Python but the push towards iterators was quite deliberate.